### PR TITLE
feat: lobby page

### PR DESCRIPTION
**Pull request by Suntoh**
## What I did

- generate room id(Math.random)
- joined 2 player to the 
- create lobby room

## Note

didn't get user information from the database
now providing the session

### CODE REVIEW

File: apps/server/.env.example
Comment (Imeany12) [inline apps/server/.env.example:1]: This file don't need to be delete krub

File: apps/client/src/app/lobby/[roomId]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
+'use client';
+import Image from 'next/image';
```
Comment (Imeany12) [inline apps/client/src/app/lobby/[roomId]/page.tsx]: Can we do this for easiness to read?
```suggestion
'use client';

import Image from 'next/image';
```

File: apps/client/src/app/lobby/page_0.tsx
Comment (Imeany12) [inline apps/client/src/app/lobby/page_0.tsx:1]: A QUESTION: what is this page for?

File: apps/server/src/server.ts [hunk 2]
```diff
@@ -61,8 +61,9 @@ io.on('connection', (socket) => {
         rooms[roomId] = [];
       }
       rooms[roomId].push([username, socket.id]);
-      socket.to(roomId.toString()).emit('update_players', rooms[roomId]);
+      console.log(rooms[roomId]);
```
Comment (Imeany12) [inline apps/server/src/server.ts:64]: A QUESTION: have you implement this? cuz I don't get this log when i join a room

**Review by Imeany12 — CHANGES_REQUESTED** [review]

File: apps/client/src/app/lobby/page_0.tsx
Comment (Suntoh) [reply apps/client/src/app/lobby/page_0.tsx:1]: I used it for debug now i can delete

File: apps/server/src/server.ts (hunk 2 above)
Comment (Suntoh) [reply apps/server/src/server.ts:64]: yes, this will log in the terminal. I have add for guest player. previously, it won't log if it's guest login.

**Review by Imeany12 — APPROVED** [review]
LGTM

**Pull request merged**